src/GitHubCreateMergePRs/config.xml

```diff
@@ -7,24 +7,21 @@
     <!-- Roslyn branches (non-vs-deps) (flowing between releases) -->
     <merge from="release/dev16.3" to="release/dev16.4" />
     <merge from="release/dev16.4" to="release/dev16.5" />
-    <merge from="release/dev16.5" to="release/dev16.6-preview2" />
-    <merge from="release/dev16.6-preview2" to="master" />
-    <merge from="master" to="release/dev16.7-preview1" />
+    <merge from="release/dev16.5" to="release/dev16.6" />
+    <merge from="release/dev16.6" to="master" />
```

Comment:
dev16 schedule says there will be preview3, preview4, and preview5. does this naming still make sense in light of that?